packedForest/test/test.cpp

```diff
@@ -22,3 +22,4 @@
 #include "fpTests/fpRerFBaseTest.h"
 #include "fpTests/binnedBaseTest.h"
 #include "fpTests/fpForestTest.h"
+#include "fpTests/fpTwoMeanSplitTest.h"
```

Comment:
The tests you added in unsupervised folder don't seem to have been added here (and thus won't be run)